github/data_source_github_organization.go

```diff
@@ -1,10 +1,11 @@
 package github
 
 import (
+	"context"
 	"log"
 	"strconv"
 
-	"github.com/google/go-github/v36/github"
+	"github.com/google/go-github/v35/github"
```

Comment:
```suggestion
	"github.com/google/go-github/v36/github"
```

Can we get this aligned to the latest version the project is using?